SAMv1.tex

```diff
@@ -344,28 +353,30 @@ \subsection{The alignment section: mandatory fields}
   \item Sum of lengths of the {\tt M/I/S/=/X} operations shall equal
     the length of {\sf SEQ}.
   \end{itemize}
-\item {\sf RNEXT}: Reference sequence name of the primary alignment of the NEXT read in the
-  template. For the last read, the next read is the first
-  read in the template. If {\tt @SQ} header lines are present, {\sf
-    RNEXT} (if not `*' or `=') must be present in one of the {\tt SQ-SN}
-  tag. This field is set as `*' when the information is unavailable, and
-  set as `=' if {\sf RNEXT} is identical {\sf RNAME}. If not `=' and the
-  next read in the template has one primary mapping (see also bit
-  0x100 in {\sf FLAG}), this field is identical to {\sf RNAME} at the primary line of the
-  next read.  If {\sf
-    RNEXT} is `*', no assumptions can be made on {\sf PNEXT} and bit
+\item {\sf RNEXT}: Reference sequence name of the NEXT read in this
+  template alignment, where NEXT is defined to be the next read in
+  template coordinates rather than mapping coordinates.  For the last
+  read, the next read is the first read in this template alignment.
+  Multiple template alignments may exist, with RNEXT/PNEXT forming a
```

Comment:
Where supp alignment fit in should be explicitly stated. This is an issue with the current specs as it doesn't explicitly state that RNEXT and PNEXT should be to the non-supp primary record.

We should explicitly state that RNEXT and PNEXT never point to supp alignment records.